src/main/java/seedu/address/logic/parser/DeleteCommandParser.java

```diff
@@ -1,29 +1,51 @@
 package seedu.address.logic.parser;
 
+import static seedu.address.logic.Messages.DELETE_COMMAND_USAGE;
 import static seedu.address.logic.Messages.MESSAGE_INVALID_COMMAND_FORMAT;
+import static seedu.address.logic.Messages.MESSAGE_MISSING_INDEX;
 
 import seedu.address.commons.core.index.Index;
 import seedu.address.logic.commands.DeleteCommand;
+import seedu.address.logic.commands.DeleteContactCommand;
 import seedu.address.logic.parser.exceptions.ParseException;
 
 /**
- * Parses input arguments and creates a new DeleteCommand object
+ * Parses input arguments and creates a new DeleteContactCommand object
```

Comment:
Should it be 'DeleteCommand object' instead of the extended class?

Applies to line 17 as well.

Reply:
deletecommand is an abstract class so it no longer can be created as an object. But I made a mistake here as it is too specific to contact when the comment should be more general like "subclass of deletecommand". I will rectify this. thank you